### PR TITLE
Language rework

### PR DESCRIPTION
**Pull request by comradekingu**
"Open source" translates to a term not trademarked by the OSI in languages other than English.

### CODE REVIEW

**Comment by tflidd** [comment]
> "Open source" translates to a term not trademarked by the OSI in languages other than English.

I didn't get this problem. Why can't we use the term "open source"?

The rest is much better to read :+1:

**Comment by comradekingu** [comment]
@tflidd It can be used, sure. However in translation it will mean the equivalent of "the source is open, in some manner", possibly only for reading. As some non-OSI open source projects/licenses are like this, (like the Vivaldi browser) it creates confusion as to the value that Nextcloud holds in addition to this by also bringing freedom 0, 2 3 and 4. In effect calling NC open source is selling it short.

**Comment by jospoortvliet** [comment]
I like it. Not every sentence, but it is a language improvement over my... marketing-speak ;-)

I think I will merge it and then let Jan & Marie edit it a bit further, there are a few changes they wanted that conflict with the changes you did. @jancborchardt @shironextcloud what do you think?

File: page-include.php [hunk 1]
```diff
@@ -18,7 +18,7 @@
 		<div class="row">
 			<div class="col-md-6 topheader">
 				<h1><?php echo $l->t('Nextcloud Include');?></h1>
- 				<h2><code><?php echo $l->t('Supporting diversity in open source');?></code></h2>
+ 				<h2><code><?php echo $l->t('Supporting diversity, in and by way of, libre software');?></code></h2>
```
Comment (jancborchardt) [inline page-include.php]: That new sentence is too convoluted and not understandable, sorry.

File: page-include.php [hunk 2]
```diff
@@ -28,7 +28,7 @@
     <div class="container">
         <div class="row revealOnScroll">
             <div class="col-md-8 col-md-offset-2">
-                <p class="section--paragraph text-center"><?php echo $l->t('Diversity strengthens a community. The Nextcloud Include initiative helps underrepresented groups to join the Nextcloud project, building an inclusive and diverse space where we collaborate and develop world class software.');?></p>
+                <p class="section--paragraph text-center"><?php echo $l->t('Diversity strengthens Nextcloud and its community. Include helps everyone partake in development thereof.');?></p>
```
Comment (jancborchardt) [inline page-include.php]: "Partake" and "thereof" is very old-style English. Remember that a lot of people of underrepresented groups are not native speakers. The old sentence is better, sorry. :/

File: page-include.php [hunk 3]
```diff
@@ -99,9 +98,9 @@
...
-		<p class="section--intro text-center revealOnScroll"><?php echo $l->t('We encourage people from underrepresented groups to apply for these positions and to sent their resume.');?></p>
-		<p class="section--intro text-center revealOnScroll"><?php echo $l->t('You can find open positions on our jobs page but please don\'t hesitate to send you resume if there is nothing that fits you - we just might not yet know we need you!');?></p>
+		<p class="section--intro text-center revealOnScroll"><?php echo $l->t('Nextcloud is always on the look-out for new and hard found talent, and we\'d like to give beginning and aspiring developers, marketeers and new people a chance to learn practical skills with us.');?></p>
+		<p class="section--intro text-center revealOnScroll"><?php echo $l->t('If your talent is underrepresented, apply for open positions with your resume by going to the jobs page, or invent a position by reaching out in any other way.');?></p>
```
Comment (camilasan) [inline page-include.php]: People are under represented, not their talents.

File: page-include.php [hunk 4]
```diff
@@ -119,16 +118,15 @@
...
+                <p class="section--paragraph__tittle"><?php echo $l->t('What is the Nextcloud community, and why are you not in it?');?></p>
+                <p class="section--paragraph"><?php echo $l->t('We are a very nice, we think, friendly and welcoming. Like a cat picture. Saying "you are welcome to join the conference" without providing a playground for parents is not enough, pro-activity is the order of the day.');?></p>
+                <p class="section--paragraph"><?php echo $l->t('Libre software is built on meritocracy, meaning capability is ideally the only barrier, try as we might, we fail sometimes at that. Even software autists (ahum) are human, and we can\'t help taking non-technical factors, like other people, into account when judging code. The resulting self-duality fails at valuing some contributions, even if equal or of better quality than others. Don\'t think you don\'t fit in, we are trying not to. The best thing we can do is extend in-group preference to everyone wishing to help, and by extention, the people you bring in to that effect. In terms of general society, we are different from it, and wish to expand the good sides of this with you.');?></p>
```
Comment (camilasan) [inline page-include.php]: Over complicated to only say that meritocracy doesn't work ;-)

File: page-include.php [hunk 5]
```diff
@@ -119,16 +118,15 @@
...
-                <p class="section--paragraph"><?php echo $l->t('Did you know our website is entirely built in github including this page? Find a typo - you can propose an improvement using the same workflow as we use to build our software. The same goes for our documentation and translation, those, too, are collaborated on in the open.');?></p>
+                <p class="section--paragraph"><?php echo $l->t('And still, you are the undisputed champion of our minds. Nextcloud is certainly no less open now than before we announced Include, and this is your special chance at being an untraditional champ.');?></p>
+                <p class="section--paragraph"><?php echo $l->t('Mentorship, travel support and internships have always been available to anyone active in our community and will continue to be.');?></p>
+                <p class="section--paragraph__tittle"><?php echo $l->t('Is Nextcloud Include only for developers, and/or people that like cat pictures?');?></p>
```
Comment (camilasan) [inline page-include.php]: Ok, I don't think you are taking those changes seriously.

File: page-include.php (hunk 4 above)
Comment (comradekingu) [reply page-include.php]: @camilasan I am not sure what you got from what I wrote, or what your take on it is, but this is no way to acknowledge contributions.

File: page-include.php (hunk 1 above)
Comment (comradekingu) [reply page-include.php]: Depends what you mean by "diversity", which is my initial quandary to set out to solve.

File: page-include.php (hunk 2 above)
Comment (comradekingu) [reply page-include.php]: The idea is to eventually translate this, which any and all takers are free to translate. I think you will agree that makes it in turn more accessible to a broader audience.
"Underrepresented" begs the question of how. This is not answered.
I understand some groups, even at single factor analysis, are unrepresented.

**Comment by comradekingu** [comment]
@tflidd Open Source™, in languages other than English (where the term is not trademarked), means nothing other than the source being open, which is indistinguishable from for example the Vivaldi project, where the source is open/available, but the project is not free/libre software. It also encompasses non-commercial projects, like the old MAME license.

@camilasan Who are "we", and what is "sexist/racist/…" about tech, and how does it relate to Nextcloud? Meritocracy worked just fine for me when I started contributing to it.

File: page-include.php (hunk 5 above)
Comment (comradekingu) [reply page-include.php]: The idea was to champion new ideas where ideas are lacking. The specifics of which seem to implicitly be that Nextcloud has had no specific consideration for inclusion prior to Include.

The change is as I see it from employing non-science, meaning bias, to compensate for what is undisclosed as "structural disadvantage". If no group preference has existed before, why will a undisclosed model of representation help matters? It is still very much internal to the thinking of Nextcloud, and even to me, as a current contributor, completely unclear.
What groups of people see representation at Nextcloud now?
What groups of people are "underrepresented", and why is it a structural disadvantage?

File: page-include.php (hunk 3 above)
Comment (comradekingu) [reply page-include.php]: Do I understand correctly that want to represent people by other means than the credence of their character, which has hitherto been their ability and willingness to contribute?
Does it relate to handicaps only, and if so, I don't understand the notion of "sexist", as you stated elsewhere.

**Comment by tflidd** [comment]
> @tflidd Open Source™, in languages other than English (where the term is not trademarked), means nothing other than the source being open, which is indistinguishable from for example the Vivaldi project, where the source is open/available, but the project is not free/libre software. It also encompasses non-commercial projects, like the old MAME license.

but "libre" is just the French word for open. And for *BSD license standards, we are not really open enough. So I'd still use the open source term (understandable for 99% of the users) and perhaps add the license for experts.



> @camilasan Who are "we", and what is "sexist/racist/…" about tech, and how does it relate to Nextcloud? Meritocracy worked just fine for me when I started contributing to it.

It's really interesting to see the debates about something, most people agree on. But the actual wording is quite difficult and if you want, you can misunderstand a lot:
I suppose all people at Nextcloud are nice people, so idiots are people from underrepresented group...



> @camilasan Who are "we", and what is "sexist/racist/…" about tech, and how does it relate to Nextcloud? Meritocracy worked just fine for me when I started contributing to it.

If you are not part of such a group, it is hard to see their needs and worries. And the limit with meritocracy is that it depends a lot what the current developers think of your contribution. So something like the "Web Accessibility for Screen Reader Users" might be less likely to merge, because it is harder to test or people need to know how to test it first.

In the end, it is a text like for nearly all companies have but the real actions are more important.

**Comment by camilasan** [comment]
@camilasan Who are "we", and what is "sexist/racist/…" about tech, and how does it relate to Nextcloud? Meritocracy worked just fine for me when I started contributing to it.

Sure, you are a white man. So take my word when I say: tech is not nice for non white males. 
Please, respect our decision of not taking your changes. As you just pointed out, you do not understand or see the problem and @tflidd pointed out why.

There is a problem, it is 2018, please, look it up yourself. Tons of research, statistics and anecdotes prove it. We won't debate that, it's like arguing with a flat-earther.

**Comment by shironextcloud** [comment]
@comradekingu, as I've already stated I believe that your rewording doesn‘t reflect the communication style we go for at Nextcloud.
We - the people who are in charge of building the Include program - all agreed on rejecting this pull request. The way we worded things reflects what we want to do.
I'd really appreciate it if you could respect that.

**Comment by jancborchardt** [comment]
@comradekingu Regarding background reading on research, articles and the issue with the current state, see the [articles section on Open Source Diversity](https://opensourcediversity.org/#articles).

Fully agree with @camilasan and @shironextcloud. And as @tflidd said:

> If you are not part of such a group, it is hard to see their needs and worries.

**Comment by comradekingu** [comment]
@tflidd Libre in French has no bearing on "libre" in English, "libre software" is a [recognized term](https://en.wikipedia.org/wiki/Free_software) to mean the same as "free software", without the duality it shares with "gratis", even technically different from freeware, the mistake is easy to make. One thereby needs only avoid the term "free software", in English. In languages where this duality does not exist, one can freely translate "libre" to ones equivalent of "Frei" in German. This is exactly what the BSD license, two or three clause, is. An additional expressed copyleft freedom atop that can be added on where it applies, for A/GPLv3+ NC software.

You present a functional argument of inclusion, which illustrates the point. It is unclear. I could take a gander at whether people agree or not if I knew that the argument was.

Current developers knew neither my age, sex, economic status, age or sexual preference when I started contributing to Nextcloud. Only the bulk of prior work, which I assume was not looked into in great detail, and even so represented no other premise, and is well within the scope of meritocracy.
Did this screen accessibility scenario actually happen? NC has AAA rating for accessibility, as per its own manuals.

I work for many different tenants of industry and tech, regardless of ethics, they all have the same awards, the same text, and the same distribution of employees in categories mentioned.

@camilasan To first deal with stating there is a problem, of current year. What exactly is the problem? What problem would I be looking to find proof of? Me asking questions of this calibre, is not analogous to being a flat-earther. I find this attempted offence awkward. The question does not go away when derided. Neither do I.

I implore you to read, https://opensourcesurvey.org/2017/ as linked to on https://opensourcediversity.org/#articles

> "rudeness (45% witnessed, 16% experienced), followed by name calling (20% witnessed, 5% experienced) and stereotyping (11% witnessed, 3% experienced)."

Here you can make a direct impact on the openness and friendliness in libre software. You do not do the committee of Include any favours by championing this behaviour.

To call my efforts 
> Over complicated ranting
 was already on the indelicate side, this is unwarranted.

To deal with the rest, I see you have assumed and invoked my characteristics in some manner you can claim no apparent insight, and if not for a proposed belief that they may inherently hold virtue, are to remain irrelevant. As a good-faith actor, I will take such a precaution, as I want you to represent your views as best you can.

@tflidd Did not point anything out to me in regards to my question, I am obviously neither a sexist, nor racist, so what was pointed out was merely an observation of perceived difference, should I, in it being stated as a rhetorical question, not have undergone the proposed hardship of being anything other, I assume that to mean privileged. Assigning such virtue without evidence is supremacist. In that same token of saying out-group knows out-group experience, it is equally valid to say anyone else can not truly know privilege. This is the logical extension of not depriving anyone of an opinion on account of their race and gender, which is a view I assume you to be sympathetic to. This all of course amounts to only a limitation of free-expression that goes both ways, and is no way to debate. This stewardship of race and gender is one that has no validity other than anecdotal, it pushes stereotypes, and is fundamentally lacking in empathy. An outward projection not founded in a consensus.

@jancborchardt I am of course non the wiser as to my original question.
You thumbed up
```
camilasan 6 days ago

People are under represented, not their talents.
```
What does that mean?

Having read all articles on OSD it gave no insight into what "diversity" constitutes, in Include usage. The use of the term is as encompassing in its use, as it is broad in its application. For the purposes of understanding what it means in practice, my English comes up short.

No model of diversity was presented, other than in the case of this [census on part of Mozilla](https://opensource.com/article/17/9/diversity-and-inclusion-innovation) of race, gender, seniority, technical background and language. I am no expert in British law, but I imagine the first two would have to have been given voluntary. The statistics no less seem in tune with what I have observed there.

If current studies in this field is anything to go by, Germany will see fewer legible applicants coming out of academia in the coming years, as has already been the trend with post-post industrial societies. Female STEM applicant percentage dropping: http://www.europarl.europa.eu/RegData/etudes/IDAN/2015/510025/IPOL_IDA(2015)510025_EN.pdf

It is illegal to discriminate on grounds of gender in Germany, as per the German General Act on Equal Treatment (Allgemeines Gleichbehandlungsgesetz, AGG).
https://blogs.orrick.com/employment/2018/08/06/the-impact-of-the-third-gender-for-employers-in-germany/ Possibly if you don't present for example a job listing as gender neutral, you can be penalized, after a potential law lands 1st of Jan 2019, for not including all judicially observed genders, which is a guarantee that they will be gender neutral, and that much of what constitutes these articles will actually fall to that basic premise.

http://www.europarl.europa.eu/RegData/etudes/IDAN/2015/510025/IPOL_IDA(2015)510025_EN.pdf

> KEY FINDINGS
> 
> Equality  between  men  and  women  in  the  labour  market  is  guaranteed  by  law; 
> however, de facto equality between women and men in gainful occupation has 
> not yet  been  realised in  Germany  and  strong  gender  specific  differences still  prevail in the labour market.

To what degree, and end-goal, the reader is left to ascertain.

> A multitude of initiatives have campaigned for binding  rules  in  order  to  change  the  gender-related  composition  of  corporate  boards. 
> Positive  experiences  in  this  field  especially  in  Norway  motivated  stakeholders  to  require gender  quotas in  order  to  increase  the number  of  women  on  boards."
> … “We are not willing to continue to tolerate the violation of Article 3 Paragraph 2 Basic Law

Which reads (translated to English):
> (2) Men and women shall have equal rights. The state shall promote the actual implementation of equal rights for women and men and take steps to eliminate disadvantages that now exist.

Nowhere was equity mentioned, you can read equality as equality of opportunity, ensuring no equity outcome.

The trend is regressing vis a vis earlier years, or stagnant. The numbers from the open source diversity links don't take into account technical employment or not, of which many jobs in tech naturally don't fall under.

---

It is my view that considerations for gender and race in the hiring process is illegal in Germany, that the procurement of such statistics is potentially illegal, and that the use of quotas in this field is not legal. NC realizing it has a bias, or that it may have one, for reasons undisclosed, makes it no more apt at acting out a solution, rather than further enforcing it as a means of original bias, unless NC can rationalize it, or even what the bias is.

It stands to reason to ask what the actual actions are going to amount to, when the language is this vague. Whatever the sought group of applicants, others will read it, and maybe as I, find meritocracy attacked without even so much as a slither of compelling evidence to do so.

**Comment by jospoortvliet** [comment]
This is not a discussion about wording, but a discussion about the essence of what Nextcloud Include is. If you disagree with us, so be it, but you obviously don't want to understand what we are trying to do here. You are free to start another program to encourage people to join open source, what we do with Include - encourage people from underrepresented groups to join our community - is not up for debate.
Again, as @shironextcloud said, please respect the decision that was made.

**Pull request closed without merging**